modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java

```diff
@@ -68,9 +77,40 @@ public void deleteRelatedModel(
 				deletionType,
 				ObjectRelationshipConstants.DELETION_TYPE_CASCADE)) {
 
+			ExecutorService executorService = new ThreadPoolExecutor(10, 10);
```

Comment:
Hi @pedro-oliveira446, can we use the 
`ExecutorService executorService =
			SystemExecutorServiceUtil.getExecutorService();` ?
On SystemExecutorServiceUtil, the poolSize and coreSize is already configured.

Reply:
Hey @linolaoj,

In my local environment, every time I tested it with the current code, it worked as expected. I attempted to implement the change you requested, but the results were inconsistent, and I wasn't able to pinpoint the exact cause.